src/homework_1/queens_problem.py

```diff
@@ -0,0 +1,32 @@
+BOARD_SIZE = 8
+
+
+class BailOut(Exception):
```

Comment:
here you have 2 empty lines between constant and code, earlier you put 0 lines. Styling should be consistent.